builder/virtualbox/ovf/step_import.go

```diff
@@ -42,7 +42,7 @@ func (s *StepImport) Cleanup(state multistep.StateBag) {
 
 	driver := state.Get("driver").(vboxcommon.Driver)
 	ui := state.Get("ui").(packer.Ui)
-	config := state.Get("config").(*Config)
+	config := state.Get("config").(Config)
```

Comment:
I think that's a great idea!! I will make the changes

Reply:
We just merged a bunch of fixes so that the config is always being put into state as a pointer, so this is probably now broken the other way. I do agree that if you don't need to write back to the config in this step, and the config is only being used for a couple of static options, it would be a nice improvement to add the fields to the step structure and stop pulling the config from state here.